browser/resources/settings/default_brave_shields_page/default_brave_shields_page.js

```diff
@@ -72,7 +72,7 @@ Polymer({
 
     Promise.all([this.browserProxy_.getAdControlType(), this.browserProxy_.isFirstPartyCosmeticFilteringEnabled()])
         .then(([adControlType, hide1pContent]) => {
-      this.adControlType_ = (adControlType === 'allow' ? 'allow' : (hide1pContent ? 'block' : 'block_third_party'))
+      this.adControlType_ = (adControlType ? 'allow' : (hide1pContent ? 'block' : 'block_third_party'))
```

Comment:
This looks great but nit, maybe would read cleaner to just not have nested ternary statements at all

Reply:
++ from me on that too

Reply:
Changed to improve readablity. PTAL again :)

Reply:
nailed it :)